.gitignore

```diff
@@ -1,4 +1,5 @@
 *.rs.bk
+.*sw*
```

Comment:
`[._]*.sw[a-p]` is a better solution for matching vim swap files.

Reply:
will amend